tests/test_changed.py

```diff
@@ -236,7 +266,7 @@ def test_output_removed_code_in_function(
     their names will also be detected.
     """
     diff = MagicMock()
-    diff.diff = GIT_DIFF_CODE_REMOVED_IN_FUNCTION
+    diff.diff = git_diff
```

Comment:
This used to mask the run with GIT_DIFF_CHANGE_IN_FUNCTION. Fixing this required me to change the data in GIT_DIFF_CHANGE_IN_FUNCTION to have 'test_class_two_test_one' method have a change.